draft-ietf-cbor-7049bis.md

```diff
@@ -1014,7 +1014,7 @@ format is invalid.
 ### Epoch-based Date/Time {#epochdatetimesect}
 
 Tag number 1 contains a numerical value counting the number of seconds
-from 1970-01-01T00:00Z in UTC time to the represented point in civil
+from 1970-01-01T00:00Z in UTC time to the represented point in civil (clock)
 time.
```

Comment:
That change does not work for me.  Civil time is a term of art, and all relevant civil time scales are now based on UTC.
Wikipedia has the nice definition of civil time: "statutory time scales designated by civilian authorities", but also cops out at "local time indicated by clocks".
So if we want to explain "civil time", we need to include the whole thing (or, point to a source, which might even be WIkipedia).

Reply:
I'll revert this. Geez, people get twitchy talking about dates and times...